packages/contracts/contracts/voting/majority/IMajorityVoting.sol

```diff
@@ -180,35 +230,41 @@ interface IMajorityVoting {
     /// @return The participation value.
     function participation(uint256 _proposalId) external view returns (uint256);
 
-    /// @notice Returns all information for a proposal by its ID.
+    /// @notice Returns the vote mode stored in the plugin settings.
+    /// @return The vote mode parameter.
+    function voteMode() external view returns (VoteMode);
+
+    /// @notice Returns the support threshold parameter stored in the plugin settings.
+    /// @return The support threshold parameter.
+    function supportThreshold() external view returns (uint64);
+
+    /// @notice Returns the minimum participation parameter stored in the plugin settings.
+    /// @return The minimum participation parameter.
+    function minParticipation() external view returns (uint64);
+
+    /// @notice Returns the minimum duration parameter stored in the plugin settings.
+    /// @return The minimum duration parameter.
+    function minDuration() external view returns (uint64);
+
+    /// @notice Returns the minimum voting power required to create a proposa stored in the plugin settings.
+    /// @return The minimum voting power required to create a proposal.
+    function minProposerVotingPower() external view returns (uint256);
```

Comment:
This may force us to do workarounds later on with OVOTE, even if it is just returning 0. Probably the best option anyway

Reply:
OVOTE doesn't need to inherit from `IMajorityVoting`. However, I think a `minProposerVotingPower > 0` requiring a  vote weight of at least 1 (or even 100, no matter where it comes from) can make sense in the OVOTE case.
Since these are just plugins, this can be reworked later on. Let's keep it for now as is

Reply:
After discussing this last Friday, we decided that the common interface will be limited to events.